db/badgerdb/badger.go

```diff
@@ -48,6 +48,7 @@ func (tx ReadTx) Discard() {
 
 // Get implements the db.WriteTx.Get interface method
 func (tx WriteTx) Get(k []byte) ([]byte, error) {
+	//lint:ignore S1016 TMP
```

Comment:
@mvdan I've added this *linter ignore* comment as staticcheck was complaining for the `ReadTx{tx: tx.tx}` of the next line. Same for line 72.

Reply:
That lint error seems valid though. You can just do `ReadTx(tx).Get(k)`.

Reply:
Great, thanks!